packages/recs/src/Entity.ts

```diff
@@ -29,3 +29,18 @@ export function createEntity(
 
   return entity;
 }
+
+/*
+ * Get the symbol corresponding to an entity's string ID.
+ * Entities are represented as symbols internally for memory efficiency.
+ */
+export function getEntitySymbol(entityString: string): EntitySymbol {
+  return Symbol.for(entityString) as EntitySymbol;
```

Comment:
we could, but do we need to? Even if something else outside of MUD is using the same symbol, I think that shouldn't be an issue. If we don't need to I'd like to avoid string manipulations here for performance reasons (eg. we apply `getEntityString` for every symbol when calling `world.getEntities()`, so any additional operation we do in there might cause issues if we're handling lots of entities)

Reply:
ah good point, there isn't much of an issue with overlaps